README.md

```diff
@@ -28,15 +28,14 @@ dist/omniSearch.js
 Run this to update files
 
 ```
-yarn
-yarn dev
+npm install
+npm run dev
 ```
 
 ### Coding standards
 
 Run prettier via the following command:
 
 ```shell
-    docker run -it --volume ${PWD}:/app --rm node:20 yarn --cwd /app install
-    docker run -it --volume ${PWD}:/app --rm node:20 yarn --cwd /app prettier src --fix --write
+    npm run coding-standards-apply
```

Comment:
One of your comments disappeared here. I removed you docker suggestion because it did not work. I also removed my own because that did not work either.

Reply:
```shell
docker compose build
docker compose run --rm php npm install
docker compose run --rm php npm run coding-standards-apply
```

works fine (for me) – and even changes something in `src/omniSearch.js`.